### PR TITLE
Splits into sub-commands

### PR DESCRIPTION
**Pull request by kimpepper**

### CODE REVIEW

File: cmd/create/common.go [hunk 1]
```diff
@@ -0,0 +1,35 @@
+package create
+
+import (
+	"context"
+	"github.com/alecthomas/kingpin"
```
Comment (nickschuch) [inline cmd/create/common.go]: Split the deps

File: cmd/create/deployment.go [hunk 2]
```diff
@@ -0,0 +1,53 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"github.com/alecthomas/kingpin"
```
Comment (nickschuch) [inline cmd/create/deployment.go]: Split the deps

File: cmd/create/deployment.go [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+}
+
+func (v *cmdDeployment) run(c *kingpin.ParseContext) error {
+	requiredContexts := make([]string, 0, 5)
+	if len(v.contexts) > 0 {
+		requiredContexts = v.contexts
+	}
+
+	deploymentRequest := &github.DeploymentRequest{
+		Ref:              github.String(v.ref),
```
Comment (nickschuch) [inline cmd/create/deployment.go]: WHY GITHUB!!!!!!!!!!

File: cmd/create/status.go [hunk 4]
```diff
@@ -0,0 +1,49 @@
+package create
+
+import (
+	"context"
+	"fmt"
+	"github.com/alecthomas/kingpin"
```
Comment (nickschuch) [inline cmd/create/status.go]: Split the deps

**Comment by nickschuch** [comment]
Looks awesome!

File: cmd/create/deployment.go (hunk 3 above)
Comment (kimpepper) [reply cmd/create/deployment.go]: Which part?

Comment (kimpepper) [reply cmd/create/deployment.go]: Oh. Having it's own String function. 🤔

**Pull request merged**